src/ad-bidders/prebid/index.ts

```diff
@@ -620,4 +630,18 @@ export class PrebidProvider extends BidderProvider {
 	calculatePrices(): void {
 		return;
 	}
+
+	setMaximumAdSlotHeight(slotName: string): void {
+		const slotRefresherConfig = context.get('slotConfig.slotRefresher.sizes');
+
+		if (!slotRefresherConfig) return;
+
+		const slotHeightLimit = slotRefresherConfig[slotName][1];
+
+		if (!slotHeightLimit) return;
+
```

Comment:
```suggestion
		const slotRefresherConfig = context.get('slotConfig.slotRefresher.sizes');

		if (!slotRefresherConfig) return;

		const slotHeightLimit = slotRefresherConfig[slotName][1];

		if (!slotHeightLimit) return;

```
selected part is duplicated. Maybe it can be extracted into parent class? leaving only something like "getSlotHeightLimit" with check is it empty or not? Lets try to not increase size of file and reduce code duplication. It will help us in near-future task focusing on size reduction and code splitting :)